### ISSUE TITLE
[Bug] The skeleton loading view doesn't match with design

### ISSUE THREAD

**Issue by minhnimble** [bigdreamer1610/SurveyApp-Swift#13]
## Issue

The application needs to show the initial skeleton loading view with animation after first entering the app with a valid logged in session like design. Currently there are a few problems with the initial skeleton loading screen:
- The skeleton loading view doesn't animate and just stay static.
- The color and corner radius of the skeleton items doesn't match with the design and there is no gradient feeling.

https://github.com/bigdreamer1610/SurveyApp-Swift/assets/70877098/17aee46f-52e7-4fbc-8fbd-a3d03f9216ca

## Expect

The skeleton loading view needs to look like the following design with proper animation while it is being shown:

<img src="https://user-images.githubusercontent.com/70877098/159680292-df2c41e9-49af-4a0f-bc86-2b71cfe69433.png" width="300">

**Comment by bigdreamer1610** [comment]
do I need to implement the black background when loading too?

**Comment by minhnimble** [comment]
Yes, the expectation is that skeleton loading view needs to look like the design.

**Comment by bigdreamer1610** [comment]
Fixed on #20
